utils/dbfixup.py

```diff
@@ -87,7 +87,7 @@ def add_zero_bits(tile_type):
             print(line, file=f)
 
```

Comment:
Okay, true. I'm not sure how this relates to the changes I've done in this PR. This sounds like an unrelated issue. (It would be helpful to add comments to this file regardless of if we merge this patch.)